packages/core/src/App/Components/Elements/NotificationMessage/__tests__/notification.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Notification from '../notification.jsx';
+
+describe('Notification component', () => {
+    it('should render the NotificationBanner component when "data.type" is "news"', () => {
+        render(<Notification data={{ type: 'news' }} />);
+        const element = screen.getByTestId('notification-banner');
```

Comment:
could you please here also change testid naming with dt_ and snakecase.
check please other cases

Reply:
yes sure